packages/widgets/tests/src/commandpalette.spec.ts

```diff
@@ -686,9 +686,6 @@ describe('@lumino/widgets', () => {
         it('should create the full class name for the item node', () => {
           let name = renderer.createItemClass({ item, indices: null, active: false });
           let expected = 'lm-CommandPalette-item testClass';
-          /* <DEPRECATED> */
-          expected = 'lm-CommandPalette-item testClass';
```

Comment:
Looks like this was a typo to begin with?

Reply:
Yeah both of these look like typos in the initial PR.